### PR TITLE
Up docs

### PR DESCRIPTION
**Pull request by vincenthoogsteder**
Updated the getting started with Up docs page and some minor tweaks on home docs.

### CODE REVIEW

File: docs/docs/home/up.md [hunk 1]
```diff
@@ -31,64 +26,82 @@ docker-compose up -d
...
+	a dashboard view.
+
+If you run anything other than Metabase, this will also work with the same flow for any SQL-speaking BI tool.
+
+### Generating Events
+To generate event data yourself, simply go to the local version of the [Objectiv website](http://localhost:8080/) and click around. When you trigger an event, you can see a request show up in your docker logs. If you used a terminal to spin up the containers, it shows there as well. 
+Both notebooks will show the new events if you rerun any of the cells.
+
+### Instrument your own website or app
+You can seamlessly switch Objectiv Up from the demo website to your own website or app; the steps to take:
```
Comment (borft) [inline docs/docs/home/up.md:87]: Switching back, after truncating the table is not really possible as the data is gone ;) I think we should also add a note to mention that restarting the stack (specifically the objectiv_notebook container) will result in resetting the database. That means any changes in the data are lost.

Comment (vincenthoogsteder) [reply docs/docs/home/up.md:87]: can you pls rewrite this piece so it a good overview of steps-to-take? Don't have enough detail to do that myself

File: docs/docs/home/up.md [hunk 2]
```diff
@@ -31,64 +26,82 @@ docker-compose up -d
...
+1. Clean up the demo data
+```bash 
+echo "TRUNCATE data" | docker exec -i objectiv_postgres psql -U objectiv
+```
+2. Instrument your app with the [Objectiv tracker](/tracking/introduction.md).
+3. Configure the tracker to use the Collector by setting the endpoint to http://127.0.0.1:8081; here is 
+	[an example](https://objectiv.io/docs/tracking/react-native/how-to-guides/getting-started#configuring-the-plugin) 
+	on how to do that for React Native. 
+
```
Comment (borft) [inline docs/docs/home/up.md:96]:
```suggestion
You can seamlessly switch Objectiv Up from the demo website to your own website or app; the steps to take:
1. Clean up the demo data
```bash 
echo "TRUNCATE data" | docker exec -i objectiv_postgres psql -U objectiv
```
2. Instrument your app with the [Objectiv tracker](/tracking/introduction.md).
3. Configure the tracker to use the Collector by setting the endpoint to http://127.0.0.1:8081; here is 
	[an example](https://objectiv.io/docs/tracking/react-native/how-to-guides/getting-started#configuring-the-plugin) 
	on how to do that for React Native. 


:::note
Restarting the notebook container will reset the database, and restore the demo data.
:::
	
```

File: docs/docs/home/up.md [hunk 3]
```diff
@@ -31,64 +26,82 @@ docker-compose up -d
...
+### Instrument your own website or app
+You can seamlessly switch Objectiv Up from the demo website to your own website or app; the steps to take:
+1. Clean up the demo data
+```bash 
+echo "TRUNCATE data" | docker exec -i objectiv_postgres psql -U objectiv
+```
+2. Instrument your app with the [Objectiv tracker](/tracking/introduction.md).
+3. Configure the tracker to use the Collector by setting the endpoint to http://127.0.0.1:8081; here is 
+	[an example](https://objectiv.io/docs/tracking/react-native/how-to-guides/getting-started#configuring-the-plugin) 
+	on how to do that for React Native.
```
Comment (borft) [inline docs/docs/home/up.md:95]:
```suggestion
1. Clean up the demo data
```bash 
echo "TRUNCATE data" | docker exec -i objectiv_postgres psql -U objectiv
```
2. Instrument your app with the [Objectiv tracker](/tracking/introduction.md).
3. Configure the tracker to use the Collector by setting the endpoint to http://127.0.0.1:8081; here is 
	[an example](https://objectiv.io/docs/tracking/react-native/how-to-guides/getting-started#configuring-the-plugin) 
	on how to do that for React Native. 

:::note
Restarting the notebook container will reset the database, and restore the demo data.
:::
```

**Pull request merged**